Tests/InstallationTests/SPMCustomEntitlementComputationInstallation/fastlane/README.md

```diff
@@ -0,0 +1,267 @@
+fastlane documentation
```

Comment:
probably don't need all the fastlane stuff since we can access through the root fastfile right?

Reply:
This was autogenerated, probably because it inherits stuff through the `import`: https://github.com/RevenueCat/purchases-ios/blob/53e3f083d635e9365a1f8618f8d63a035e58f0f4/Tests/InstallationTests/SPMCustomEntitlementComputationInstallation/fastlane/Fastfile